app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb

```diff
@@ -155,14 +128,21 @@ app.controller('RecipeFinderCtrl', ['$scope', '$http', '$routeParams', function(
     }
   };
 
+  $scope.searchWord = "Ingredient";
```

Comment:
this is to initially set the selector to the ingredient option.... therefore later the if statement is to differentiate whether the user searches by ingredient or keyword